job_bundles/povray-3.7/template.yaml

```diff
@@ -0,0 +1,117 @@
+specificationVersion: "jobtemplate-2023-09"
+name: "POV-Ray 3.7 Render Job"
+description: "POV-Ray raytracing render job template with configurable parameters"
+
+parameterDefinitions:
+  - name: InputScene
+    type: PATH
+    objectType: FILE
+    dataFlow: IN
+    description: "Input POV-Ray scene file (.pov)"
+    default: "sample/sample.pov"
```

Comment:
nit: Could add `*.pov` file filter hints for the GUI (similar to what https://github.com/aws-deadline/deadline-cloud-samples/blob/mainline/job_bundles/blender_render/template.yaml does)